src/core/net/nat64_translator.hpp

```diff
@@ -379,6 +381,8 @@ class Translator : public InstanceLocator, private NonCopyable
 
     ProtocolCounters mCounters;
     ErrorCounters    mErrorCounters;
+
+    State mState;
```

Comment:
Let's init the `mState` from `Translator` constructor.

Reply:
Done